pkg/apis/beat/v1beta1/beat_types.go

```diff
@@ -13,8 +13,9 @@ import (
 
 // BeatSpec defines the desired state of a Beat.
 type BeatSpec struct {
-	// Type is the type of the Beat to deploy (filebeat, metricbeat, heartbeat, etc.). Any string can be used,
-	// but well-known types will be recognized and will allow to provide sane default configurations.
+	// Type is the type of the Beat to deploy (filebeat, metricbeat, heartbeat, auditbeat, journalbeat, packetbeat, etc.).
+	// Any string can be used, but well-known types will be recognized and will allow to use the correct image and
```

Comment:
maybe "but well-known types..."

can be "but well-known types will have the image field defaulted and have the appropriate Elasticsearch roles created automatically"?